kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -19,13 +19,7 @@
         <first id="bundle.file">
             <union>
                 <fileset dir="${project.build.directory}/plugins">
-                    <include name="${bundle.name}_${bundle.version}*.jar" />
-                </fileset>
-                <fileset dir="../target-definition/common/source/plugins">
-                    <include name="${bundle.name}-${bundle.version}*.jar" />
-                </fileset>
-                <fileset dir="${kura.osgi.repo}/plugins/">
-                    <include name="${bundle.name}-${bundle.version}*.jar" />
+                    <include name="${bundle.name}*.jar" />
```

Comment:
This could match multiple bundles if their name has a common prefix, we could at least add '-' or '_' (the one that is correct) after the name and make the build fail if the fileset contains more than one entry

Reply:
Addressed in [408a2fc](https://github.com/eclipse-kura/kura/pull/5850/commits/408a2fcd284f63f12ed25c96a67b80f5ae680f65)